### DISCUSSION TITLE
Hardware recommendations JeelizFaceFilter

### DISCUSSION THREAD

**Discussion by sociuswalter** [jeeliz/jeelizFaceFilter#297] [General]
Hello, 
i tried to run the facefilters on a raspberry 3b+ but they wouldn't work i think due to poor performance.
Would they run on a raspberry 4 or in general what would be the hardware recommendations?

Thank you

**Comment by xavierjs** [comment]
Hi,

What matter is not the CPU computing power but the GPU computing power.
First, please check by looking at chrome://gpu that WebGL is hardware accelerated (not Swiftshader).

I suggest you use Nvidia Jetson hardware. It is like Rpi but GPU oriented. They are more expensive but it is really worth it. 
You can take a look at Jetsonjs ( https://github.com/jeeliz/jetsonjs ), I ran Jeeliz libs on Nvidia Jetson TX2 in 2018.
Now they are also less expensive jetson like the nano (100USD).

I offer development services to implement Chrome/WebGL stack to run Jeeliz libs on Nvidia Jetson, you can contact me on linkedin for more information: https://www.linkedin.com/in/xavierbourry/

**Comment by sociuswalter** [comment]
Thank you for the fast reply, i will take some time to "digest" the input.

**Comment by sociuswalter** [comment]
I activated GL in rpi3b+ via raspi-config. Now even the Camera stream isn't shown. So i guess the gpu performance is too poor.
In the next week(s?) i will test with a friends RPI 4. 
Again thanks for the advice.
 I will definitely come back to you on linkedin when i take the project from piloting to a professional level.
